gorm/plugin/tracing/tracing.go

```
@@ -87,8 +121,9 @@ var operationNameKey struct{}
 func (t *tracing) before(operationName string) func(*gorm.DB) {
 	return func(tx *gorm.DB) {
 		tx.Statement.Context, _ = t.provider.StartSpan(
-			context.WithValue(tx.Statement.Context, operationNameKey, operationName),
-			operationName,
+			context.WithValue(tx.Statement.Context, operationNameKey,
```

golangci-lint (1.62.2)

SA1029: should not use empty anonymous struct as key for value; define your own type to avoid collisions

(staticcheck)